src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -391,53 +392,58 @@ std::optional<units::length::inch_t> DragonLimelight::EstimateTargetXDistance()
     std::optional<int> aprilTagID = GetAprilTagID();
     if (!aprilTagID)
     {
-        // d=(h2-h1)/tan(a1+a2)
-        double tangent = units::math::tan(mountingAngle + targetPitch.value());
-
-        if (abs(tangent) < 0.01)
-        {
-            return std::nullopt;
-        }
-        else
+        if (targetPitch)
         {
-            units::length::inch_t estimatedTargetDistance = (m_noteVerticalOffset - mountingHeight) / tangent;
-
-            return estimatedTargetDistance;
+            double tangent = units::math::tan(mountingAngle + targetPitch.value());
+
+            if (abs(tangent) < 0.01)
```

Comment:
i had to change this one to std:abs instead of abs as I don't think cabs handled the double and just returned 0.